<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>complaint_mode_set_dialog</class>
 <widget class="QDialog" name="complaint_mode_set_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stiffness set</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>K(N/mm,deg):</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stiffness_X_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stiffness_Y_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stiffness_Z_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rx:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stiffness_Rx_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ry:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stiffness_Ry_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stiffness_Rz_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="stiffness_set_pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>111</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>B(N/mm/s,deg):</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="damp_X_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="damp_Y_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="damp_Z_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rx:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="damp_Rx_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ry:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="damp_Ry_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="damp_Rz_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
